Options.lua

```diff
@@ -22,6 +22,9 @@ local GetCVar, GetCVarBool, GetCVarDefault = GetCVar, GetCVarBool, GetCVarDefaul
 local UnitsExists, UnitName = UnitsExists, UnitName
 local GameTooltip = GameTooltip
 local GetSpellInfo = Addon.GetSpellInfo
+local GetAddOnEnableState = (C_AddOns and C_AddOns.GetAddonEnableState)
```

Comment:
There is a type here. Correct is C_AddOns.GetAddOnEnableState (Addon => AddOn)

Reply:
Fixed.